views/workouts.handlebars

```diff
@@ -1,31 +1,29 @@
 {{#if workouts.length}}
```

Comment:
It looks like more is changed here than actually is... 
a few of the id and data tags were adjusted - closing /each moved up a few lines to fix the event listener issue